gammastep: add page (#12196)
tldr-pages/tldr: pages/common/gammastep.md

# Gammastep

> > Adjust the color temperature of your screen according to time of day.

Reviewer: ```suggestion
> Adjust the color temperature of your screen according to the time of day.
```

> > Adjust the color temperature of your screen according to the time of day.

Reviewer 2: ```suggestion
> Adjust the screen's color temperature according to the time of day.
```
We usually avoid using words like "you" or "your"

> More information: <https://gitlab.com/chinstrap/gammastep>.

> - Turn on Gammastep with 5700K [t]emperature during day and 3600K at night:

Reviewer: ```suggestion
- Turn on Gammastep with 5700K [t]emperature during the day and 3600K at night:
```

> `gammastep -t {{5700}}:{{3600}}`

Reviewer 3: Imo it should still be a placeholder because placeholders mark the part that can be changed by the user. Instead we should change the description.

> `gammastep -t {{5700}}:{{3600}}`

Author: I certainly intended for the placeholders to be placeholders rather than explicit examples. One moment for an update...

> `gammastep -t {{5700}}:{{3600}}`

Author: I also applied this suggestion to screen brightness, although that edit makes the value range description more cumbersome. 

It raises a new question: is it better to be more concise (i.e. not specify value ranges) than to include the generic placeholder -- or instead better to replace specific placeholder examples with generics?

> `gammastep -t {{5700}}:{{3600}}`

Reviewer 3: > It raises a new question: is it better to be more concise (i.e. not specify value ranges) than to specify the values of the generic placeholder -- or instead better to replace specific examples with generic placeholders?

Imo that's a question of taste. Afaik the tldr project does not have a policy on that. I personally think that it is better to have specific examples (such as this page) because this automatically resolves questions about e.g. the required data type, format etc.

> I ask partially because screen brightness now seems longer than other command descriptions I've used in TLDR; and partially because this PR is setting up a general review of color-shifting commands. The suggestions from this review increase divergence between gammastep and redshift

I'll change the redshift page. I personally think that the descriptions are not too long.

> `gammastep -t {{5700}}:{{3600}}`

Author: @sebastiaanspeck - what do you think? I believe your review is the last element of this PR.

- Turn on Gammastep with a manually specified custom [l]ocation:

`gammastep -l {{latitude}}:{{longitude}}`

> - Turn on Gammastep with 70% screen [b]rightness during day and 40% brightness at night (between 0.1 and 1):

Reviewer: ```suggestion
- Turn on Gammastep with 70% screen [b]rightness during the day and 40% brightness at night (between 0.1 and 1):
```

> - Turn on Gammastep with 5700K (t)emperature during day and 3600K at night:
> 
> `gammastep -t {{5700}}:{{3600}}`
> 
> - Turn on Gammastep with a manually specified custom (l)ocation:
> 
> `gammastep -l {{latitude}}:{{longitude}}`
> 
> - Turn on Gammastep with 70% screen (b)rightness during day and 40% brightness at night (between 0.1 and 1):
> 
> `gammastep -b {{0.7}}:{{0.4}}`
> 
> - Turn on Gammastep with custom (g)amma levels (between 0 and 1):
> 
> `gammastep -g {{red}}:{{green}}:{{blue}}`
> 
> - Turn on Gammastep with a c(O)nstant unchanging color temperature:
> 
> `gammastep -O {{temperature}}`

Reviewer 3: ```suggestion
- Turn on Gammastep with 5700K [t]emperature during day and 3600K at night:

`gammastep -t {{5700}}:{{3600}}`

- Turn on Gammastep with a manually specified custom [l]ocation:

`gammastep -l {{latitude}}:{{longitude}}`

- Turn on Gammastep with 70% screen [b]rightness during day and 40% brightness at night (between 0.1 and 1):

`gammastep -b {{0.7}}:{{0.4}}`

- Turn on Gammastep with custom [g]amma levels (between 0 and 1):

`gammastep -g {{red}}:{{green}}:{{blue}}`

- Turn on Gammastep with a c[O]nstant unchanging color temperature:

`gammastep -O {{temperature}}`
```

Author: Got it. Thanks!
